packages/core/test/financial-templates/expiring-multiparty/PricelessPositionManager.js

```diff
@@ -939,251 +939,614 @@ contract("PricelessPositionManager", function(accounts) {
     // No excess collateral after all have settled.
     await expectNoExcessCollateralToTrim();
   });
+  describe("Financial Product Library", () => {
```

Comment:
introducing this extra describe added an extra indent through this set of tests and totally broke the git diff.